{"name": "test_get_user", "status": "failed", "statusDetails": {"message": "AssertionError: assert 401 == 200\n +  where 401 = <Response [401]>.status_code", "trace": "api_client = <api.api_client.APIClient object at 0x000002068F1917F0>\n\n    @allure.feature(\"API Feature\")\n    def test_get_user(api_client):\n        endpoint = '/auth/user'\n        response = api_client.get(endpoint)\n    \n>       assert response.status_code == 200\nE       assert 401 == 200\nE        +  where 401 = <Response [401]>.status_code\n\ntests\\test_api.py:27: AssertionError"}, "start": 1722718924709, "stop": 1722718925102, "uuid": "dd66fa7f-6061-47be-8779-223200d7ae73", "historyId": "4a50f1871faa930645ada9579db37bad", "testCaseId": "4a50f1871faa930645ada9579db37bad", "fullName": "tests.test_api#test_get_user", "labels": [{"name": "feature", "value": "API Feature"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "DESKTOP-1RESUV2"}, {"name": "thread", "value": "9380-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}